clients/mediaconvert.go

```diff
@@ -434,7 +438,7 @@ func getTargetDir(url *url.URL) string {
 			dir = ""
 		}
 	}
-	return dir
+	return path.Join(requestID, dir)
```

Comment:
Exactly, here! Why doesn't this work?
```suggestion
	return path.Join(dir, requestID)
```